disco_aws_automation/disco_elasticsearch.py

```diff
@@ -25,8 +25,9 @@ class DiscoElasticsearch(object):
     """
 
     def __init__(self, environment_name, config_aws=None, config_es=None,
-                 route53=None):
+                 config_vpc=None, route53=None):
         self.config_aws = config_aws or read_config()
+        self.config_vpc = config_vpc or read_config('disco_vpc.ini')
```

Comment:
Nit-pick: `disco_vpc.ini` could be replace by `CONFIG_FILE` in `disco_vpc.py` to avoid duplication.